### ISSUE TITLE
option to turn off "click on folder stickers to change it"

### ISSUE THREAD

**Issue by dsimonow** [Make-md/makemd#177]
The option to click on the stickers of folders to change them is great but it is too aggressive. 
Accidental clicks are common and as soon as the file structure gets deeper it is almost impossible not to click constantly on change stickers.
A setting to turn that quick access off would help.

**Comment by Peeeet** [comment]
I would agree.  Actually might even suggest that clicking on the icon in the tree view never brings up the change dialog but instead you can only change it by clicking on the larger icon at the top of the view of the note page itself.  Maybe a right click menu entry for cases where this might not be the case (canvas or when not viewing folder-notes with contexts turned off)
